pkg/config/config.go

```diff
@@ -1094,6 +1094,7 @@ func InitConfig(config Config) {
 	bindEnvAndSetLogsConfigKeys(config, "runtime_security_config.activity_dump.remote_storage.endpoints.")
 	config.BindEnvAndSetDefault("runtime_security_config.event_stream.use_ring_buffer", false)
 	config.BindEnv("runtime_security_config.event_stream.buffer_size")
+	config.BindEnvAndSetDefault("runtime_security_config.envs_whitelist", []string{})
```

Comment:
Maybe we could provide a nice set of defaults (like `LD_PRELOAD` etc)

Reply:
I'm fine adding default values like `LD_PRELOAD`, but shouldn't we add them directly to the config file for clarity

Reply:
I would rather use `runtime_security_config.envs_with_value`

Reply:
We should maybe update the config template file like for the sensitive words https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L1565-L1575

Reply:
Done